flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -174,6 +175,15 @@ public Builder table(Table newTable) {
       return this;
     }
 
+    /**
+     * Set the write properties for Flink sink. Write properties (with the same keys) defined in table properties
+     * will be overwritten by properties provided here.
+     */
+    public Builder properties(Map<String, String> newProps) {
```

Comment:
Other APIs use `set` for individual key/value properties and `setAll` for a map of properties. Can you update this to `set` or `setProperties`? It isn't clear what `properties` does.

In the `Parquet` write builder, for example, we have these methods:

```java
    public WriteBuilder set(String property, String value) {
      config.put(property, value);
      return this;
    }

    public WriteBuilder setAll(Map<String, String> properties) {
      config.putAll(properties);
      return this;
    }
```